build.gradle.kts

```diff
@@ -112,6 +113,14 @@ signing {
     sign(publishing.publications)
 }
 
+tasks.withType<Sign>().configureEach {
```

Comment:
> Looks like these changes were pushed accidentally.

It disables signing task when property `signing.skip` is `true`. Code taken from [plugin docs, Example 7](https://docs.gradle.org/current/userguide/signing_plugin.html#ex-specifying-when-signing-is-skipped).

Reply:
I changed the question. Sorry!

Reply:
> Can we implement such functionality without additional plugin?

`net.saliman.properties` just allows not to specify additional command line property each time running a task (i.e. ` -Psigning.skip=true`).

Reply:
It's just because different environments for this project look over-needed.
It would be great for some service which can be deployed on Test/Stage/Prod.
Here - it's just library without all these things. 
Let's try to find a simples way!

Reply:
> Let's try to find a simples way!

The simplest way is to specify `isRequired`:
```kotlin
signing {
    sign(publishing.publications)
    isRequired = false
}
```
... but the task will silently fail if signatory config in pipeline would be misconfigured for some reason.

Reply:
Without plugin I have to configure each `publish_TargetPlatform_PublicationToMavenLocal` task with one of the options listed [here](https://docs.gradle.org/current/userguide/project_properties.html#sec:project_properties)? for example:

![image](https://github.com/Scogun/kcron-common/assets/9965210/096e7ace-431b-4138-b23f-d11a104b70b3)

Reply:
Anyway, I'll could revert this commit and use own separate branch for this purpose.

Reply:
@Myshkouski you again pushed "skip" code.
You can use `git add -i` command to add only specific lines of changed code instead of whole files.